packages/finoaconnect/README.md

```diff
@@ -0,0 +1,39 @@
+# @web3-onboard/finoaconnect
+
+## Wallet module for connecting FinoaConnect SDK to web3-onboard
+
+#### Install
+
+`npm i @web3-onboard/core @web3-onboard/finoaconnect`
+
+## Usage
+
```

Comment:
```suggestion

Optional initialization object
```typescript
/** Optional object provided to the initiation of the wallet connector. 
 * When not included, the wallet connector service connects to FinoaConnect production systems.
 * @field {url} URL of the FinoaConnect backend systems to be used
 * @field {labelSuffix} arbitrary string label to denote the context of the URL field  */
export interface FinoaWalletOption {
  url?: string
  labelSuffix?: string
}
```

```